at-syntax/atsyntax.gpt

```diff
@@ -1,4 +1,5 @@
 Name: At syntax filtern
 Description: Translates @name command into "Call tool name with command"
+Type: context
```

Comment:
i think this one is not a context? i think its used as an input filter in the tool.gpt.

Reply:
You're right. Removed.